### PR TITLE
Update README to v0.16.1

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
made with `./hack/prepare-release.sh`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,4 +48,4 @@ $ kubectl get no -o json | jq '.items[].metadata.labels'
 
 ```
 
-[documentation]: https://kubernetes-sigs.github.io/node-feature-discovery
+[documentation]: https://kubernetes-sigs.github.io/node-feature-discovery/v0.16
```
Comment (TessaIO) [inline README.md:51]: why did you change this? Don't you think that it's better to redirect to a stable version?

Comment (ArangoGutierrez) [reply README.md:51]: yeah is the script that change that bit... grrrr one second

Comment (ArangoGutierrez) [reply README.md]: done
